README.md

```diff
@@ -1,6 +1,9 @@
 
 # NodeSchool Vancouver 🇨🇦
 
+- On the web: https://nodeschool.io/vancouver/
+- On Gitter: [nodeschool/organizers](https://gitter.im/nodeschool/organizers)
```

Comment:
That Gitter is not active. My suggestion would be to not point folks at resources where they may not be heard and feel ignored. 

Folks do have conversations in the discussions on the [meetup.com group](https://www.meetup.com/nodeschool-vancouver) and organizers respond there.

```suggestion
- On Gitter: [nodeschool/organizers](https://gitter.im/nodeschool/organizers)
```